kazoo/hosts.py

```diff
@@ -17,7 +18,10 @@ def collect_hosts(hosts, randomize=True):
         if host is None:
             raise ValueError("bad hostname")
         port = int(res.port) if res.port else 2181
-        result.append((host.strip(), port))
+
```

Comment:
So after thinking about this I am wondering how RRsets that grow or shrink would be handled here. It seems once this is called now its relying on python code to do the name resolution, whereas previously it was relying on dns to correctly round-robin on reconnect... Am I wrong in saying that resolving the dns hostname at this time would basically make the client itself unable to take advantage of any changes in a hostnames A records that happen after the fact...